docs/manual/forms/examples/form_with_geometry.rst

```diff
@@ -0,0 +1,115 @@
+.. _example_form_with_geometry:
+
+===============================================
+Kaarmateriaal en registratie in de Objecten API
+===============================================
+
+In dit formulier maken we gebruik van de "Kaart"-component om coördinaten van een
+locatie uit te vragen en te registreren. Het formulier wordt gekoppeld met de
+`Objecten API`_ voor het registreren van de inzendingsgegevens.
+
+We gaan er van uit dat de koppeling met de Objecten API
+:ref:`geconfigureerd <configuration_registration_objects>` is, en gebruiken als
+objecttype-URL ``https://objecttype-example.nl/api/v2/objecttype/123``.
+
+Formulier maken
+===============
+
+We bouwen een formulier met twee kaarten - één kaart wordt gebruikt voor "dé" geometrie
+van het hele Object in de Objecten API en de andere kaart wordt als extra/vrije-vorm
+attribuut meegestuurd.
+
+#. Maak een formulier aan met de volgende gegevens:
+
+   * **Naam**: Kaartgegevens registreren in de Objecten API
+
+#. Klik op het tabblad **Stappen en velden**.
+#. Klik aan de linkerkant op **Stap toevoegen** en selecteer **Maak een nieuwe
+   formulierdefinitie**.
+#. Onder de sectie **(Herbruikbare) stapgegevens** vul het volgende in:
+
+   * **Naam**: Kaarten
+
+#. Scroll naar beneden en klik de sectie **Speciale velden** aan.
+#. Sleep een **Kaart** component op het witte vlak en vul de volgende
+   gegevens in:
+
+   * **Label**: Hoofdgeometrie
+
+#. Klik vervolgens de **Registratie** tab aan en vul de volgende gegevens in:
+
+   * **Registratie-attribuut**: Locatie > Coördinaten
+
+#. Druk daarna op **Opslaan**.
+
+#. Sleep een tweede **Kaart** component op het witte vlak, vul de volgende
+   gegevens in en druk daarna op **Opslaan**:
+
+   * **Label**: Extra geometrie
+
+#. Klik op het tabblad **Registratie**.
+#. Klik op **Registratiepluginoptie toevoegen** en vul de volgende gegevens in:
+
+    * **Naam**: Objecten API
+    * **Registratiemethode**: Objecten API registratie
+    * **Objecttype**: ``https://objecttype-example.nl/api/v2/objecttype/123``
+    * **JSON-inhoud sjabloon**:
+
+      .. code-block:: django
+
+          {
+              "extraGeometrie": : {% as_geo_json variables.extraGeometrie %}
```

Comment:
Is the second `:` correct?
Also: why a template tag and not a filter? To support other projections?

Reply:
it is not

there was some back and forth with a generic `{% as_json %}` tag which turned out to be not desired after all, because we want to apply geojson wrapping to the coordinates array.

I don't think that it was a conscious decision to not use a filter, but the pattern of custom tags is followed and I think I like that consistency. Filters are also a bit harder to understand/build if you're not familiar with the django template language, so for simplicity's sake this is probably better.

I do think that tags have a compile/validate advantage over filters which are expected to crash silently, while tags are expected to blow up spectacularly.